server/src/main/java/org/elasticsearch/search/DocValueFormat.java

```diff
@@ -246,6 +246,22 @@ public double parseDouble(String value, boolean roundUp, LongSupplier now) {
             return parseLong(value, roundUp, now);
         }
 
+        @Override
```

Comment:
We use DocValueFormat as part of a hash key now (so that a lookup can access the same field with different formats and still use caching); this was the only impl with no `hashCode()/equals()`